CML_SCAN.podspec

```diff
@@ -0,0 +1,13 @@
+
+Pod::Spec.new do |s|
+  s.name         = "CML_SCAN"
+  s.version      = "1.0.0"
+  s.summary      = "This is a Chanel internal framework that simplifies the implementation of barcode scanner."
+  s.homepage     = "https://sources.clasp-infra.com/tfs/DefaultCollection/Mobile/CML/_git/CML_SCAN"
+  s.license      = "Chanel"
+  s.author             = { "SandhuSukhjeet" => "sukhjeetsingh.s@solcen.in", "NSubramanya" => "subramanya.n@solcen.in" }
```

Comment:
Alignment in this file is inconsistent. If you want to vertically line up the `=`, do that, but do so on all lines. Otherwise, put the `=` right after the LHS.